packages/jackett/build.sh

```diff
@@ -0,0 +1,43 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/jackett/jackett"
+TERMUX_PKG_DESCRIPTION="API Support for your favorite torrent trackers"
+TERMUX_PKG_LICENSE="GPL-3.0"
```

Comment:
I had a look at the file structure of the finished package and noticed that:
- 1.) The license file is present in both `$PREFIX/share/doc/jackett/copyright` and `$PREFIX/lib/jackett/LICENSE`.
Only the first one should exist.
- 2.) This can be changed to `GPL-2.0-or-later`, since we recently made the build system distinguish between different variants of the GPL and the license used by Jackett is the GPLv2 with the "-or-later" clause included.
#23478

```suggestion
TERMUX_PKG_LICENSE="GPL-2.0-or-later"
```